client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -138,41 +138,47 @@
                 <div
                   v-for="(f, filterIndex) in filterGroup.filter"
                   :key="filterIndex"
-                  class="d-flex align-items-center gap-1 rounded px-2 py-1 text-nowrap bg-light"
+                  class="active-filter px-2 py-1 bg-light"
                 >
-                  <span>
+                  <p class="active-filter-text">
                     {{ formatActiveFilterName(f) }}
```

Comment:
Operators should be same as they are in the dropdown of UI.
Also the LIKE operator is not handled!